packages/components/checkbox/src/Checkbox.tsx

```diff
@@ -64,10 +74,27 @@ export const Checkbox = forwardRef<HTMLButtonElement, CheckboxProps>(
           aria-describedby={description}
           aria-invalid={isInvalid}
           onCheckedChange={handleCheckedChange}
-          {...others}
+          /**
+           * If the checkbox doesn't have any direct label (children) then we should try to
+           * get an eventual alternative label from FormField.
+           * On last resort, we shouldn't forget to define an aria-label attribute.
+           */
```

Comment:
IMO no need for this comment should be tested and explained in the tests

Reply:
Well, I think, just as `Switch`, that both could be useful. Wdyt? https://github.com/adevinta/spark/pull/832/commits/06e5a6d2af6bdc42149c1042cbedcfae2a30fee3

Reply:
We always need an Id for each element/component?

Reply:
This is just for this one, that needs an accessible name. The usecase shouldn't occur here, but it's technically possible, just as for the Switch component. So for the sake of consistency I choosed to use same approach.